rqd/tests/rqmachine_tests.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+#  Copyright (c) 2018 Sony Pictures Imageworks Inc.
```

Comment:
I've preferred a single line separating the shbang and copyright.

Reply:
Done.